pennylane/ops/identity.py

```diff
@@ -197,6 +196,8 @@ def pow(self, _):
         return [I(wires=self.wires)]
 
 
+I.__name__ = "Identity"
```

Comment:
This way works, but I think also the following would be fine:
```python
class I(...)
    ...

    __name__ = "Identity"
```
Not sure if there any implications of one choice over the other.

Reply:
I tried that and it didnt work:
```
class asd:
    __name__ = "other_name"

asd.__name__
```

```
'asd'
```

Reply:
Thanks, interesting! Wonder why :thinking:

Reply:
I think the class variables also get initialized only when an instance is constructed for that class.